src/components/webViewRe.re

```diff
@@ -0,0 +1,289 @@
+module type WebViewComponent = {
```

Comment:
Can you extract the content into `webViewRe.rei`? The component doesn't need to be a functor. The reason that `View` is a functor, is that there is also `Animated.View` where we reuse the implementation.